app/swagger/requests/notifications.rb

```diff
@@ -53,6 +53,53 @@ class Notifications
             end
           end
         end
+
+        operation :patch do
+          extend Swagger::Responses::AuthenticationError
+          extend Swagger::Responses::ValidationError
+          extend Swagger::Responses::RecordNotFoundError
+
+          key :description, 'Update an existing dismissed status notification record'
+          key :operationId, 'patchDismissedStatus'
+          key :tags, %w[notifications]
+
+          parameter :authorization
+
+          parameter do
+            key :name, 'subject'
+            key :in, :path
+            key :description, 'The subject of the dismissed status notification (i.e. "form_10_10ez")'
```

Comment:
This could be `key :example 'form_10_10ez'`

Reply:
:point_up: nitpick

Reply:
No, it's a good point.  When I did the `GET` notification endpoint a few days ago, I tried adding `key :example,  "form_10_10ez"` in here.

And it works for the generated swagger docs as desired; however, all of the notification swagger specs blow up with this error:

https://gist.github.com/hpjaj/febee4942469e81f9eb9f68c5c1bf42b

I tried digging into it the other day, and couldn't find a solution.  I just did some more investigation, and am still coming up short.  I'll timebox some more digging.  Pls let me know if you have any ideas, too.